erpnext/patches/v12_0/update_is_cancelled_field.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import unicode_literals
+import frappe
+from erpnext.regional.india.setup import make_custom_fields
```

Comment:
Unused import? And maybe you should print the exception in te execute. None should be pass? isn't it more clear?